src/accessibility-and-localization/accessibility.md

```diff
@@ -163,7 +163,14 @@ Users can skip this step if you programmatically auto-enable
 accessibility for your app using this API:
 
 ```dart
-SemanticsBinding.instance.ensureSemantics();
+import 'package:flutter/material.dart';
+import 'package:flutter/semantics.dart';
+
+void main() {
+  WidgetsFlutterBinding.ensureInitialized();
+  SemanticsBinding.instance.ensureSemantics();
+  runApp(const MyApp());
```

Comment:
Have you tried:

```suggestion
  runApp(const MyApp());
  SemanticsBinding.instance.ensureSemantics();
```

The `WidgetsFlutterBinding` is the first call that runApp does, anyway; and I think the Semantics call needs to go *after* runApp (probably because of the WidgetsFlutterBinding call)

Reply:
/cc @yjbanov to corroborate this, he's the semantics-meister